code/03_census_and_bed_days.R

```diff
@@ -0,0 +1,868 @@
+################################################################################
+### 14. bed days data ----
+################################################################################
+
+#Get filespath+main bed days file.SAV
+
+#datafile11<-read_spss(paste0(filepath,"main bed days file.sav"))
+
+
+datafile11<-datafile %>% mutate(Healthboard=substring(Healthboard, 5)) # change nhs_board and remove the 'NHS_'
+
+hbbeddaysdiffage_grp<- datafile11 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(hbbeddaysdiffage_grp,paste0(filepath,"hb bed days diff age_grp_R.sav")) # save out file
+
+labeddaysdiffage_grp<- datafile11 %>% 
+  group_by(LocalAuthorityArea,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(labeddaysdiffage_grp,paste0(filepath,"la bed days diff age_grp_R.sav")) # save out file
+
+datafile12<-datafile11 %>% mutate(Healthboard="Scotland")
+
+Scotbeddaysdiffage_grp<- datafile12 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+
+#write_sav(Scotbeddaysdiffage_grp,paste0(filepath,"Scot bed days diff age_grp_R.sav")) # save out file
+
+
+# add files
+
+Scotlandhbbeddays<-rbind(Scotbeddaysdiffage_grp,hbbeddaysdiffage_grp)
+
+# Get Scotland and HB bed days for all ages
+
+datafile13<-Scotlandhbbeddays %>% mutate(age_grp="All")
+
+Scotlandhbbeddays<- datafile13 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+#add files
+Scotlandhbbeddaysallagesallreasons<-rbind(Scotlandhbbeddays,Scotbeddaysdiffage_grp,hbbeddaysdiffage_grp)
+
+#write_sav(Scotandhbbeddaysallagesallreasons,paste0(filepath,"Scot and hb bed days_R.sav")) # save out file
+
+datafile15<-Scotlandhbbeddaysallagesallreasons %>% mutate(reas1="All")
+
+ScotlandhbAllage_grpsbeddays<- datafile15 %>% 
+  group_by(Healthboard,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+Scotlandhbbeddaysallagegrpsallreasons<-rbind(ScotlandhbAllage_grpsbeddays,Scotlandhbbeddaysallagesallreasons)
+
+#write_sav(Scotlandhbbeddaysallagegrpsallreasons,paste0(filepath,"Scot and hb bed days all age_grps all reasons.sav")) # save out file
+
+### 15. Bed days LAs ----
+#use la bed days diff age file 
+datafile17<-labeddaysdiffage_grp %>% mutate(age_grp="All")
+table(datafile17$OBDs)
+
+labeddaysallage_grps<- datafile17 %>% 
+  group_by(LocalAuthorityArea,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+datafile18<-rbind(labeddaysallage_grps,labeddaysdiffage_grp)
+
+#write_sav(datafile18,paste0(filepath,"la bed days.sav")) # save out file
+
+datafile19<-datafile18 %>% mutate(reas1="All")
+
+labeddaysallreason_grp_high_level<- datafile19 %>% 
+  group_by(LocalAuthorityArea,age_grp,reas1) %>% 
+  summarise(OBDs=sum(OBDs,na.rm=TRUE)) %>% 
+  ungroup()
+
+datafile20<-rbind(labeddaysallreason_grp_high_level,datafile18)
```

Comment:
See "Style Guide" section in general feedback.
You use bind_rows() throughout but move to rbind() here. bind_rows() as a dplyr function should replace rbind() here.
```suggestion
datafile20 <- bind_rows(labeddaysallreason_grp_high_level, datafile18)
```